src/messages/Emote.hpp

```diff
@@ -15,6 +17,9 @@ struct Emote {
     Tooltip tooltip;
     Url homePage;
     bool zeroWidth;
+    EmoteId id;
+    EmoteAuthor author;
+    boost::optional<EmoteName> baseName;
```

Comment:
What is the baseName of an emote? Comment would be helpful for future readers here